libnetwork/libnetwork_linux_test.go

```diff
@@ -1937,8 +1941,9 @@ func TestResolvConfHost(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	if !bytes.Equal(content, tmpResolvConf) {
-		t.Fatalf("Expected:\n%s\nGot:\n%s", string(tmpResolvConf), string(content))
+	content = regexp.MustCompile("(?m)^#.*\n").ReplaceAll(content, []byte{})
+	if !bytes.Equal(content, expectedResolvConf) {
+		t.Fatalf("Expected:\n%s\nGot:\n%s", string(expectedResolvConf), string(content))
 	}
```

Comment:
Can we use `gotest.tools` here, or were we trying to avoid importing it in this package?

Reply:
I think we could use it ... I was trying to avoid getting bogged down by this gigantic (2300-line) file, all of its tests are written this way. But, will have a look at it when I get back to this PR.

Reply:
It's fine (for me) to start with the new tests (no real need to re-write existing tests, we can do that separately).

I know I avoided / removed `gotest.tools` in some packages, but that was mostly for packages that we may want to move to a separate module at some point (in which case it's nice for the module to have no, or very little dependencies). For `libnetwork`, we can assume that it's mostly an internal package (with the exception of some bits used by BuildKit), and won't be moved separate so I'm fine with having `gotest.tools` as dependency.

Reply:
Ok ... no new tests in this file, just stopping the old tests from failing on diffs in whitespace / ordering / comments. So, I've not done any rework to use gotest-stuff.